code/src/index.css

```diff
@@ -1,13 +1,207 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --primary: #ff9494;
+  --secondary: #ffd1d1;
+  --third: #ffe3e1;
+  --fourth: #fff5e4;
+  --fifth: #444649;
+  --sixth: #343437;
+}
+
 body {
-  margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
+
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  color: var(--fifth);
+
+  /* BACKGROUND ANIMATION */
+
+  background: linear-gradient(120deg, var(--primary), var(--fourth));
+  background-size: 500%;
+  animation-name: gradient;
+  animation-duration: 4s;
+  animation-iteration-count: infinite;
+  animation-direction: alternate;
+  /* https://www.youtube.com/watch?v=f3mwKLXpOLk */
+}
+
+@keyframes gradient {
+  from {
+    background-position: 0%;
+  }
+  to {
+    background-position: 100%;
+  }
+}
+
+.title {
+  font-family: "Helvetica Neue", sans-serif;
+  font-weight: 900;
+  font-size: 19px;
+  padding: 40px 0 0 5px;
+  margin: 0;
+}
+
+.sub-title {
+  margin: 5px 0 25px;
+  font-size: 14px;
+  padding-left: 5px;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.container {
+  margin: 30px auto 0;
+  width: 75%;
+}
+
+/**** FORM ****/
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-shadow: 7px 10px;
+  background-color: var(--secondary);
+  border: solid 1px var(--primary);
+  font-size: 12px;
+  color: var(--fifth);
+}
+.form {
+  width: auto;
+}
+
+.textarea-wrapper {
+  align-items: center;
+  justify-content: center;
```

Comment:
I'm not sure if `align-items` and `justify-content` work properly without `display: flexbox;` 🤔

Reply:
Ah... thank you